ResearchKit/Common/ORKFormItemCell.m

```diff
@@ -831,7 +831,7 @@ @implementation ORKFormItemScaleCell {
 }
 
 - (id<ORKScaleAnswerFormatProvider>)formatProvider {
-    if(_formatProvider == nil){
+    if (_formatProvider == nil){
         _formatProvider = (id<ORKScaleAnswerFormatProvider>)[self.formItem.answerFormat impliedAnswerFormat];
```

Comment:
But we usually don't add a space between a *cast* and the *casted value*, do we?